addons/stock/stock.py

```diff
@@ -612,7 +612,7 @@ def _price_update(self, cr, uid, ids, newprice, context=None):
         self.write(cr, SUPERUSER_ID, ids, {'cost': newprice}, context=context)
 
     def quants_unreserve(self, cr, uid, move, context=None):
-        related_quants = [x.id for x in move.reserved_quant_ids]
+        related_quants = move.reserved_quant_ids.ids
```

Comment:
Is this so critical that the change is justified?

Reply:
Yes, this saves a lot of time.